server/routes/fetchThreadForSnapshot.ts

```diff
@@ -12,15 +12,15 @@ const fetchThreadForSnapshot = async (models: DB, req: Request, res: Response, n
   if (!snapshot) return next(new Error(Errors.InvalidSnapshot));
   if (!chain) return next(new Error(Errors.InvalidChain));
 
-  const thread = await models.OffchainThread.findOne({
+  const thread = await models.OffchainThread.findAll({
     where: { 
       chain: chain,
       snapshot_proposal: snapshot,
     }
   });
   if (!thread) return res.json({ status: 'Failure', message: '' });
-
-  return res.json({ status: 'Success', result: thread.id });
+  
+  return res.json({ status: 'Success', result: thread });
```

Comment:
we'll wanna return these jsonified

Reply:
and if you only need the thread `id` and `title` in the frontend, you can just map those into objects in an array that's returned